Source/Diagnostics/BTDiagnostics.H

```diff
@@ -388,5 +398,17 @@ private:
     void RedistributeParticleBuffer (const int i_buffer);
     void UpdateVarnamesForRZopenPMD();
 
+    amrex::Real gettlab (int i_buffer) override {return m_t_lab[i_buffer];}
```

Comment:
Can you add docstrings here as well?

Reply:
this I added in Diagnostics.H 
do we repeat here. no big deal if we have to. I ll just paste those doxygen lines here